test/pihole.toml

```
@@ -1413,6 +1413,17 @@
   #     true or false
   normalizeCPU = true
 
+  # Should FTL hide warnings coming from dnsmasq?
+  #
+  # By default, FTL reports warnings coming from the embedded dnsmasq DNS server to the
+  # FTL log file. These warnings can be useful to identify misconfigurations or problems
+  # with the DNS server. However, some warnings may be harmless and can be ignored in
+  # certain setups. Enabling this setting will hide dnsmasq warnings.
```

See my other comment.

```suggestion
  # certain setups. Enabling this setting will hide all dnsmasq warnings.
```